exporters/otlp/common/src/main/java/io/opentelemetry/exporter/otlp/internal/grpc/OkHttpGrpcExporter.java

```diff
@@ -229,6 +226,18 @@ public CompletableResultCode shutdown() {
     return CompletableResultCode.ofSuccess();
   }
 
+  static boolean isRetryable(Response response) {
+    // Only retry on gRPC codes which will always come with an HTTP success
```

Comment:
@jack-berg BTW, this is unlikely to be the best behavior (i.e., 503 seems like an obvious one to retry on) - gRPC retry lives in a bubble that misses standard HTTP load balancers. But since spec doesn't mention HTTP codes for gRPC retry, went with this for now.

Reply:
Yeah that's a good point. There could be a load balancer in front of the gRPC server that gets in the way of the gRPC bubble.